docs/architecture/adr-006-support-multisig-chain-link.md

```diff
@@ -0,0 +1,320 @@
+number of them sign it to generate the signed files by using the `tx sign` command with their keys. Subsequently, one of them gathers all the signed 
+files and uses the `tx multisign` command with the raw transaction file to get the multisign file. The multisign file includes not only all public keys 
+and threshold of the multisig account but also the required number of signatures so that creating the chain-link JSON from the multisign file is possible.
+
+The whole process in code is presented below:
+```go
+// getChainLinkJSONFromMultiSign generates the chain-link JSON from the multisign file and its raw transaction file
+func getChainLinkJSONFromMultiSign(
+	cmd *cobra.Command,
+	txFile string,
+	multiSignFile string,
+	chainID string,
+	chain chainlinktypes.Chain,
+) (profilescliutils.ChainLinkJSON, error) {
+	clientCtx, err := client.GetClientTxContext(cmd)
+	if err != nil {
+		return profilescliutils.ChainLinkJSON{}, err
+	}
+
+	parsedTx, err := authclient.ReadTxFromFile(clientCtx, txFile)
+	if err != nil {
+		return profilescliutils.ChainLinkJSON{}, err
+	}
+
+	parsedMultiTx, err := authclient.ReadTxFromFile(clientCtx, multiSignFile)
+	if err != nil {
+		return profilescliutils.ChainLinkJSON{}, err
+	}
+
+	txCfg := clientCtx.TxConfig
+	txBuilder, err := txCfg.WrapTxBuilder(parsedTx)
+	if err != nil {
+		return profilescliutils.ChainLinkJSON{}, err
+	}
+
+	multiTxBuilder, err := txCfg.WrapTxBuilder(parsedMultiTx)
+	if err != nil {
+		return profilescliutils.ChainLinkJSON{}, err
+	}
+
+	txFactory := tx.NewFactoryCLI(clientCtx, cmd.Flags())
+	if txFactory.SignMode() == signingtypes.SignMode_SIGN_MODE_UNSPECIFIED {
+		txFactory = txFactory.WithSignMode(signingtypes.SignMode_SIGN_MODE_LEGACY_AMINO_JSON)
+	}
+
+	sigs, err := multiTxBuilder.GetTx().GetSignaturesV2()
+	if len(sigs) != 1 {
+		return profilescliutils.ChainLinkJSON{}, fmt.Errorf("invalid number of signature")
+	}
+	multisigSig := sigs[0]
+
+	signingData := authSigning.SignerData{
+		ChainID:       chainID,
+		AccountNumber: txFactory.AccountNumber(),
+		Sequence:      txFactory.Sequence(),
+	}
+	// the bytes of plain text
+	value, err := txCfg.SignModeHandler().GetSignBytes(txFactory.SignMode(), signingData, txBuilder.GetTx())
+	if err != nil {
+		return profilescliutils.ChainLinkJSON{}, err
+	}
+
+	addr, _ := sdk.Bech32ifyAddressBytes(chain.Prefix, multisigSig.PubKey.Address().Bytes())
+	return profilescliutils.NewChainLinkJSON(
+		profilestypes.NewBech32Address(addr, chain.Prefix),
+		profilestypes.NewProof(multisigSig.PubKey, signingtypes.SignatureDataToProto(multisigSig.Data), hex.EncodeToString(value)),
+		profilestypes.NewChainConfig(chain.Name),
+	), nil
+}
+```
+
+## Consequences
+
+### Backwards Compatibility
+
+We change `Signature` of `Proof` into `SignatureDescriptor_Data` from hex-encode string of signature bytes.
+This feature is not backwards compatible. Migrating the old chain link signature to `SignatureDescriptor_Data`
+is required.
```

Comment:
```suggestion
Since we change the `Proof#Signature` field from a hex-encode string into a `SignatureDescriptor_Data`, this feature is not backwards compatible. Migrating the old chain link signatures to `SignatureDescriptor_Data`
is required.
```